test/BuyTshirt.spec.ts

```diff
@@ -0,0 +1,44 @@
+import { $, browser } from 'protractor';
+
+describe('Buy a t-shirt', () => {
+    beforeEach(() => {
+        jasmine.DEFAULT_TIMEOUT_INTERVAL = 120000;
+    });
+
+    it('then should be bought a t-shirt', async () => {
+        await browser.get('http://automationpractice.com/');
+        await (browser.sleep(10000));
+        await $('#block_top_menu > ul > li:nth-child(3) > a').click();
+        await (browser.sleep(3000));
+        await
+            $('#center_column > ul > li > div > div.left-block > div > a.product_img_link > img').click();
+        await (browser.sleep(3000));
+        await $('#add_to_cart > button > span').click();
+        await (browser.sleep(3000));
+        await $('.button-container > a').click();
```

Comment:
in the last document version this locator was updated for `[style*="display: block;"] .button-container > a`